extensions/guacamole-auth-sso/modules/guacamole-auth-sso-saml/pom.xml

```diff
@@ -143,14 +143,19 @@
                     <artifactId>commons-codec</artifactId>
                 </exclusion>
 
+                <exclusion>
```

Comment:
Might be useful to have a comment documenting the reason for the exclusions?

Reply:
Fixed via rebase and simply removing the exclusion (no longer necessary - xmlsec actually directly depends on this version of Woodstox).